python/GafferUI/MenuBar.py

```diff
@@ -220,3 +220,28 @@ def __matchingAction( self, keySequence, menu ) :
 					return a
 
 		return None
+
+## Tablet support
+#
+# Qt creates synthetic mouse events for tablet interaction. The following
+# commit intended to better support touch screens inadvertently causes tablet
+# mouse-move events (source MouseEventSynthesizedByQt) to be ignored by the
+# menu bar.
+#
+#   https://code.qt.io/cgit/qt/qtbase.git/commit/src/widgets/widgets?id=c5307203f5c0b0e588cc93e70764c090dd4c2ce0
+#
```

Comment:
Please also add a link to https://bugreports.qt.io/browse/QTBUG-77679, so we can remove this workaround should it ever get fixed.

Reply:
Added [here](https://github.com/GafferHQ/gaffer/pull/3996/files#diff-26bfee84731c40d91fb5d1898c2f8b792094869ef2b8498a56d40a672e48e564R235-R236).